Formula/l/llama-cpp.rb

```diff
@@ -0,0 +1,30 @@
+class LlamaCpp < Formula
+  desc "LLM inference in C/C++"
+  homepage "https://github.com/ggerganov/llama.cpp"
+  url "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b2715.tar.gz"
```

Comment:
we might need to throttle the formula (many releases in short time frame)

Reply:
I guess thats fine